core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/BillingApiServiceV2Spec.scala

```diff
@@ -538,7 +554,59 @@ class BillingApiServiceV2Spec extends ApiServiceSpec with MockitoSugar {
       }
   }
 
-  "GET /billing/v2/{projectName}/spendReport" should "400 when invalid dates are given" in withEmptyDatabaseAndApiServices { services =>
+  "GET /billing/v2/{projectName}/spendReport" should "200 and return only summary information when aggregation key is not present" in withEmptyTestDatabase { dataSource: SlickDataSource =>
+    val mockSpendReportingService = mock[SpendReportingService](RETURNS_SMART_NULLS)
+    // if the API service does not parse parameters and call the spendReportingService correctly, test will fail
+    when(mockSpendReportingService.getSpendForBillingProject(any[RawlsBillingProjectName], any[DateTime], any[DateTime], any[Set[SpendReportingAggregationKeyWithSub]]))
+      .thenReturn(Future.failed(new RawlsException("parameters were not parsed correctly")))
```

Comment:
Would it be helpful to also log the said parameters that aren't parsed correctly?

Reply:
Perhaps, but I'm not sure how to get those values through the mock